pkg/manager/member/pd_failover_test.go

```diff
@@ -686,8 +722,12 @@ func oneFailureMember(tc *v1alpha1.TidbCluster) {
 		pd0: {Name: pd0, ID: "0", Health: true},
 		pd2: {Name: pd2, ID: "2", Health: true},
 	}
+
+	pvcUIDSet := make(map[types.UID]struct{})
```

Comment:
this indicates that the failure member `pd1` has 2 mounted PVCs as is set in the test function.